3회차/문현동/0_반반.py

```diff
@@ -1,3 +1,21 @@
 import sys
 
 sys.stdin = open("_반반.txt")
+
+TC = int(input())
+
+for test_case in range(1, TC + 1):
+    S = input()
+    
+    ref = True
+    
+    for s in S: # 조건은 문자열 내의 어떤 하나의 문자 개수가 4개가 안되어야 하고 그 문자는 짝수개 여야 한다.
+        if S.count(s) != 4 and S.count(s) % 2 == 0:
+            pass
+        else:
+            ref = False
```

Comment:
`count` 값이 2가 아닐 때 `False`로 값을 바꾸도록 수정해도 같은 결과가 나옵니다! 😄 
```suggestion
        if S.count(s) != 2:
            ref = False
```